README.md

```
@@ -1,38 +1,79 @@
-If you're seeing this, you've probably already done this step. Congrats!
+## About The Project
 
-```bash
-# create a new project in the current directory
-npx sv create
+This is the repository for the Questers Portal, a documentation / blog / dashboard / utils for things that are not the game itself.
 
-# create a new project in my-app
-npx sv create my-app
-```
+The portal is hosted here: https://questersportal.netlify.app/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)